salt/cloud/clouds/scaleway.py

```diff
@@ -345,25 +352,30 @@ def query(method='servers', server_id=None, command=None, args=None,
     request = __utils__["http.query"](path,
                                       method=http_method,
                                       data=data,
-                                      headers={'X-Auth-Token': token,
+                                      status=True,
+                                      decode=True,
+                                      decode_type='json',
+                                      data_render=True,
+                                      data_renderer='json',
+                                      headers=True,
+                                      header_dict={'X-Auth-Token': token,
                                                'User-Agent': "salt-cloud",
                                                'Content-Type': 'application/json'})
-    if request.status_code > 299:
+    if request['status'] > 299:
```

Comment:
What is the purpose for this change?

Reply:
There was a change in the util I think (maybe a long time ago?). This module hasn't worked in a while. checking against request['status'] seems to be pretty standard in the other providers. There is some inconsistency in testing for error, but this is probably close.

Reply:
perfect, thanks, just wanted to make sure.